src/main/scala/beam/replanning/utilitybased/ChainBasedTourVehicleAllocator.scala

```diff
@@ -74,10 +74,8 @@ case class ChainBasedTourVehicleAllocator(
       getVehicularToursSortedByStartTime(householdPlans).find(rec => rec.subtour == subtour)
 
     vehicularTours foreach { vt =>
-      if (allocateVehicles(vt)) {
-        //TODO: turn back on when using intra-household choice
-//        processAllocation(vt, plan)
-      }
+      //TODO: turn back on when using intra-household choice
+      allocateVehicles(vt)
     }
```

Comment:
is allocateVehicles(vt) doing something by itself or just a check?
if it is doing something - I assume removing TODO, because it seems to be pointing at processAllocation function call
If it is not doing anything - I assume removing the whole loop